echopype/tests/preprocess/test_preprocess.py

```diff
@@ -1,3 +1,5 @@
+import sys
```

Comment:
Here and in `test_plot.py`, I assume you're going to use the `sys` package in work you'll be adding? Otherwise, it'd not currently being used.